src/graphics/constants.js

```diff
@@ -1207,3 +1207,31 @@ semanticToLocation[SEMANTIC_ATTR15] = 15;
  */
 export const CHUNKAPI_1_51 = '1.51';
 export const CHUNKAPI_1_55 = '1.55';
+
```

Comment:
this should be in the scene/constants

Reply:
and perhaps it should be SHADERPASSTYPE_

Reply:
> this should be in the scene/constants

Don't think so. Ideally the graphics module (lit-shader, standard etc) would base all its logic only on PASSTYPE and not know anything about SHADER_ (since SHADER_ types are defined in scene).

Unfortunately right now the function that maps between SHADER_ and PASSTYPE_ is implemented in scene, so graphics must know about it, but ideally we'd implement something in future which sorts that out.

(Probably standard-material-options-builder would pass in PASSTYPE_ instead of SHADER_).

Reply:
> and perhaps it should be SHADERPASSTYPE_

I thought about that too, but actually think the correct name is probably RENDERPASSTYPE_. Both seem a bit long though?

Reply:
> render pass is used for different concept

Ahh yes I forgot about your RenderPass class.

I'll name these `SHADERTYPE_` for now (since the `SHADER_` constants should probably be `SHADERPASS_`).

> I suspect I'll be moving lit-shader, standard and all that outside of the graphics

This makes sense actually, have graphics just wrap the hardware/API. I actually wonder whether having a 'materials' module might be better than lobbing it in with scene... Anyway I'll move these to scene for now.